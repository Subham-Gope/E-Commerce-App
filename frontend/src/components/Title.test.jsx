// import { describe, test, expect } from "vitest";
// import { render, screen } from "@testing-library/react";
// import Title from "./Title";

// describe("Testing the Title.jsx file", () => {
//   test("render the title", () => {
//     // Rendering and setting up the snapshot matching
//     const { container } = render(<Title text1={`Hello`} text2={`There`} />);
//     expect(container).toMatchSnapshot();

//     // checking for the texts
//     const text1 = screen.getByText(/hello/i);
//     expect(text1).toBeInTheDocument();
//     const text2 = screen.getByText(/there/i);
//     expect(text2).toBeInTheDocument();
//   });
// });
