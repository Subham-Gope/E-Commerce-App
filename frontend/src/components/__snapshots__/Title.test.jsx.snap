// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Testing the Title.jsx file > render the title 1`] = `
<div>
  <div
    class="inline-flex gap-2 items-center mb-3"
  >
    <p
      class="text-gray-500"
    >
      Hello
       
      <span
        class="text-gray-700 font-medium"
      >
        There
      </span>
    </p>
    <p
      class="w-8 sm:w-12 h-[1px] sm:h-[2px] bg-gray-700"
    />
  </div>
</div>
`;
